src/trivialapi/fleetdm/core.py

```diff
@@ -72,6 +72,28 @@ def post(self, endpoint, data=None, version="v1"):
             return resp
 
         raise Exception(resp.status_code, resp.content)
+    
+    def _raw_put(self, endpoint, data, version):
```

Comment:
Possibly time to refactor these into a `_raw_req` that calls `requests.request` and passes the method instead of individually calling `requests.put`/`post`/`delete`/whatever. https://requests.readthedocs.io/en/latest/api/#requests.request